include/clap/helpers/host.hxx

```diff
@@ -163,7 +165,7 @@ namespace clap { namespace helpers {
          // put draft ext here
       }
 
```

Comment:
Am I reading this wrong or is it a pretty big change?

if I request host.get extension I no longer get the extension which routes to my internal virtual method for one of the mapped extensions?

Reply:
It allows the host to shortcut the clap helper for a specific extension.
I think it makes sense.

Reply:
Maybe I merged this too quickly, do you see a big issue with this change?

Reply:
Maybe we could have two methods to make it clearer if that you're concern:
- `getExtension()` which would be the fallback if the clap-helper didn't provide it
- `shortcutExtension()` which would let the subclass provide the extension to use before the helper layer.

Would that make sense?

Reply:
> It allows the host to shortcut the clap helper for a specific extension. I think it makes sense.

but what if that extension is CLAP_EXT_GUI was my thinking. Then you get nullptr if you call get extension.

I definitely mix-and-match `hostProxy.paramFoo` and  `p = hostProxy.getExtension(PARAMS); p->foo(hostProxy->_host)` in my code and expect both to kinda give the same function point.

The perhaps yeah 'underlyingExtension' or 'shortcutExtension' makes sense but I think this change will break some of my code.

Reply:
This doesn't touch the `PluginProxy` or `HostProxy`.
It is only for the `Plugin` and `Host`.

Reply:
Are you saying that you are using plugin-proxy to talk to your own plugin instance within itself?